keras_nlp/models/xlm_roberta/xlm_roberta_tokenizer.py

```diff
@@ -170,44 +169,4 @@ def from_preset(
         preset,
         **kwargs,
     ):
-        """Instantiate an XLM-RoBERTa tokenizer from preset vocabulary.
-
-        Args:
-            preset: string. Must be one of {{names}}.
-
-        Examples:
-        ```python
-        # Load a preset tokenizer.
-        tokenizer = keras_nlp.models.XLMRobertaTokenizer.from_preset(
-            "xlm_roberta_base_multi",
-        )
-
-        # Tokenize some input.
-        tokenizer("The quick brown fox tripped.")
-
-        # Detokenize some input.
-        tokenizer.detokenize(tf.constant([581, 63773, 119455, 6, 147797]))
-        ```
-        """
-        if preset not in cls.presets:
-            raise ValueError(
-                "`preset` must be one of "
-                f"""{", ".join(cls.presets)}. Received: {preset}."""
-            )
-        metadata = cls.presets[preset]
-
-        spm_proto = keras.utils.get_file(
-            "vocab.spm",
-            metadata["spm_proto_url"],
-            cache_subdir=os.path.join("models", preset),
-            file_hash=metadata["spm_proto_hash"],
-        )
-
-        config = metadata["preprocessor_config"]
-        config.update(
-            {
-                "proto": spm_proto,
-            },
-        )
-
-        return cls.from_config({**config, **kwargs})
+        return super().from_preset(cls, preset, **kwargs)
```

Comment:
You need a newline at the end of each file. Are you still having issues with the `format.sh` script?

Reply:
Hi @jbischof , I'm still addressing the comments. So work is pending.
I tend to run formatting scripts upon finishing changes for every round of review.

Should I continue with this or run it every time before commit?

Thanks.

Reply:
Sorry, didn't understand this was still WIP! Up to you on how you organize your commits 😄